src/components/about-us/constants/section-02/category-ids.js

```diff
@@ -8,8 +8,8 @@ const categoryIds = {
   engineer: 'engineer',
   designer: 'designer',
   podcast: 'podcast',
-  theReporterForKids: 'theReporterForKids',
   dataJournalist: 'dataJournalist',
+  theReporterForKids: 'theReporterForKids',
```

Comment:
Question: if key orders matters in `categoryIds`, could we modify its type to array or array of object ?

Reply:
fixed